addons/website/models/ir_ui_view.py

```diff
@@ -104,6 +134,31 @@ def _views_get(self, cr, uid, view_id, options=True, bundles=False, context=None
                     result.append(r)
         return result
 
+    def _get_assets_urls(self, cr, uid, view_id, options=True, context=None):
+        """ For a given view ``view_id``, should return:
+
+        * Path of JS files used in view 
+        * Path of CSS files used in view
+        * Path of LESS files used in view
+        """
+        assets = []
+        views = self._views_get(cr, uid, view_id, options, context)
+        customized_urls = {}
+        attachments = request.registry['ir.attachment'].search_read(request.cr, SUPERUSER_ID, [('url','=like','/custom/%')], ['url','name'], context=request.context)
+        [customized_urls.update({a['name']:a['url']}) for a in attachments]
```

Comment:
Why you don't use the comprehension dict ?
customized_urls = {a['name']: a['url'] for ...}